src/assets/i18n/cy.json

```diff
@@ -18,57 +18,57 @@
-      "responseToSignsTrafficControllers": "[CY] Response to signs / signals - Traffic controllers",
-      "responseToSignsOtherRoadUsers": "[CY] Response to signs / signals - Other road users",
-      "pedestrianCrossings": "[CY] Pedestrian crossings",
-      "positionNormalStops": "[CY] Position/normal stop",
-      "awarenessPlanning": "[CY] Awareness planning",
-      "outcomeControlledStop": "[CY] Controlled stop",
-      "reverseRightControl": "[CY] Reverse right - Control",
-      "reverseRightObservation": "[CY] Reverse right - Observation",
-      "reverseParkRoadControl": "[CY] Reverse park (road) - Control",
-      "reverseParkRoadObservation": "[CY] Reverse park (road) - Observation",
-      "reverseParkCarparkControl": "[CY] Reverse park (car park) - Control",
-      "reverseParkCarparkObservation": "[CY] Reverse park (car park) - Observation",
-      "forwardParkControl": "[CY] Forward park - Control",
-      "forwardParkObservation": "[CY] Forward park - Observation",
-      "vehicleChecks": "[CY] Vehicle checks",
-      "controlledStop": "[CY] Controlled stop",
-      "eyesightTest": "[CY] Eyesight test"
+      "controlsAccelerator": "Rheolaeth - sbardun",
+      "controlsClutch": "Rheolaeth - cydiwr",
+      "controlsGears": "Rheolaeth - gêr",
+      "controlsFootbrake": "Rheolaeth - brêc troed",
+      "controlsParkingBrake": "Rheolaeth - brêc parcio",
+      "controlsSteering": "Rheolaeth - y llyw",
+      "precautions": "Rhagofalon",
+      "ancillaryControls": "Rheolyddion atodol",
+      "moveOffSafety": "Symud i ffwrdd - diogelwch",
+      "moveOffControl": "Symud i ffwrdd - dan reolaeth",
+      "useOfMirrorsSignalling": "Defnyddio drychau - rhoi signalau",
+      "useOfMirrorsChangeDirection": "Defnyddio drychau - newid cyfeiriad",
+      "useOfMirrorsChangeSpeed": "Defnyddio drychau - newid cyflymder",
+      "signalsNecessary": "Signalau - angenrheidiol",
+      "signalsCorrectly": "Signalau - yn gywir",
+      "signalsTimed": "Signalau - amseru",
+      "junctionsApproachSpeed": "Cyffyrdd - cyflymder agosáu",
+      "junctionsObservation": "Cyffyrdd - gwyliadwriaeth",
+      "junctionsTurningRight": "Cyffyrdd - troi i’r dde",
+      "junctionsTurningLeft": "Cyffyrdd - troi i’r chwith",
+      "junctionsCuttingCorners": "Cyffyrdd - torri corneli",
+      "judgementOvertaking": "Doethineb - goddiweddyd",
+      "judgementMeeting": "Doethineb - dod i gyfarfod cerbydau",
+      "judgementCrossing": "Doethineb - croesi",
+      "positioningNormalDriving": "Lleoli ar y ffordd - gyrru arferol",
+      "positioningLaneDiscipline": "Lleoli ar y ffordd - disgyblaeth lôn",
+      "clearance": "Clirio",
+      "followingDistance": "Pellter dilyn",
+      "useOfSpeed": "Defnyddio cyflymder",
+      "progressAppropriateSpeed": "Symud ymlaen - cyflymder addas",
+      "progressUndueHesitation": "Symud ymlaen - petruso gormodol",
+      "responseToSignsTrafficSigns": "Ymateb I arwyddion ffyrdd/ signalau - arwyddion traffig",
```

Comment:
There should probably be a space before the `/` in these translations e.g. `"Ymateb I arwyddion ffyrdd / signalau - arwyddion traffig"`

Reply:
Also, I would question why the Welsh translations don't use the same case. e.g.

English is "Response to signs / signals - Traffic signs"

Welsh is "Ymateb I arwyddion ffyrdd/ signalau - arwyddion traffig" but why not "Ymateb I arwyddion ffyrdd/ signalau - **A**rwyddion traffig" (capital "A")

Reply:
Not sure the rules on  capitalisation are the same in welsh.  I did see an article that said they are to an extent - but less things are capitalised and thought it best not to second guess and just cut and paste exactly as provided.

Reply:
Added space before /  as looks better, and some of them had the space in already.